packages/orchestrator-ui-components/src/rtk/endpoints/startComboBoxOptions.ts

```diff
@@ -0,0 +1,96 @@
+import {
+    ProductDefinition,
+    StartButtonOptionsResult,
+    StartComboBoxOption,
+    WorkflowDefinition,
+    WorkflowTarget,
+} from '@/types';
+
+import { orchestratorApi } from '../api';
+
+const workflowOptionsQuery = `
+    query StartOptions {
+        workflows(first: 1000000, after: 0, filterBy: [{ field: "target", value: "${WorkflowTarget.CREATE}"}]) {
+            page {
+                name
+                description
+                products {
+                    productId
+                    name
+                }
+            }
+        }
+    }
+`;
+
+const taskOptionsQuery = `
+    query StartOptions {
+        workflows(first: 1000000, after: 0, filterBy: [{ field: "target", value: "${WorkflowTarget.SYSTEM}"}]) {
+            page {
+                name
+                description
+            }
+        }
+    }
+`;
+
+export type StartButtonOptionsResponse = {
+    startComboBoxOptions: StartComboBoxOption[];
```

Comment:
To support my other comments in this file. The type 'StartComboBoxOption' is a typical view type and should not exist in the part of code that does the actual communication with the backend.